resources/views/components/slideover/global.blade.php

```diff
@@ -0,0 +1,26 @@
+{{--
+    This is a wrapper for the global slideover. Use this as much as possible to reduce the amount of html elements in the DOM.
+
+    Standard usage may look like this:
+    <x-rapidez::slideover.global :title="__('My slideover')">
+        <x-slot:label class="rounded p-3 border">
+            Click me
+        </x-slot:label>
+
+        <div class="font-semibold">
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+            Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
+            Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
+            Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+        </div>
+    </x-rapidez::slideover.global>
+--}}
+
+@props(['title', 'position' => 'left'])
+@slots(['label', 'contents'])
+
+<global-slideover title="{{ $title }}" position="{{ $position }}" contents="{{ $slot->toHtml() }}" v-slot="slideover">
```

Comment:
Can't we use a slot for the content? This feels/looks weird with HTML in it

Reply:
That's essentially what the `global-slideover-wrapper` is for. If we want to use it, we can do something like this:

```blade
@props(['title', 'position' => 'left'])
@slots(['label', 'contents'])

<global-slideover title="{{ $title }}" position="{{ $position }}" v-slot="slideover">
  <div {{ $attributes->class('contents') }}>
    <div class="hidden global-slideover-wrapper">
      {{ $slot }}
    </div>
    <label {{ $label->attributes->class('global-slideover-label cursor-pointer') }} for="slideover-global" v-on:click="slideover.open">
      {{ $label }}
    </label>
  </div>
</global-slideover>
```

However, in this case we have a pretty weird structure which isn't necessarily any better. I tried to get vue slots to work, but I wasn't able to. I could give it another go though as that would definitely be the cleanest.

Reply:
The issue with using Vue slots at least is that you can't really turn a `this.$scopedSlots.xxxxx()` into actual html, there is no helper function for it and it only returns something that Vue uses internally to generate the html.

So then what remains is the other option I posted above.